specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```
@@ -0,0 +1,2851 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Service",
+    "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.0",
+    "version": "2022-02-01-preview"
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "description": "A subscription key for a Language service resource.",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "common.json#/parameters/Endpoint"
+      }
+    ]
+  },
+  "paths": {
+    "/:analyze-text": {
+      "post": {
+        "summary": "Request text analysis over a collection of documents.",
+        "description": "Submit a collection of text documents for analysis.  Specify a single unique task to be executed immediately.",
+        "operationId": "AnalyzeText",
+        "consumes": [
+          "application/json",
+          "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "description": "Collection of documents to analyze and a single task to execute.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextTask"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
```

Is this field supposed to be an empty string? How about:
```suggestion
            "description": "A successful call results",
```

---

This was probably something I just missed.  I've just added the description here.